packages/datadog-plugin-azure-functions/src/index.js

```diff
@@ -35,11 +36,19 @@ class AzureFunctionsPlugin extends TracingPlugin {
         'aas.function.trigger': mapTriggerTag(methodName)
       }
     }, false)
```

Comment:
Unrelated to this PR, but passing `ctx` here instead of false would remove the need to explicitly assign `currentStore` and `parentStore` to the context at the end of the function.